tests/e2e/operationstorm.go

```diff
@@ -196,10 +211,17 @@ var _ = ginkgo.Describe("[csi-block-vanilla] [csi-block-vanilla-parallelized] Vo
 		ginkgo.By("Verify all volumes are accessible in the pod")
 		for index := range persistentvolumes {
 			// Verify Volumes are accessible by creating an empty file on the volume
-			filepath := filepath.Join("/mnt/", fmt.Sprintf("volume%v", index+1), "/emptyFile.txt")
-			_, err = framework.LookForStringInPodExec(namespace, pod.Name,
-				[]string{"/bin/touch", filepath}, "", time.Minute)
-			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			if windowsEnv {
+				filepath := filepath.Join("C:\\mnt\\", fmt.Sprintf("volume%v", index+1), "\\data.txt")
+				_, err = framework.LookForStringInPodExec(namespace, pod.Name,
```

Comment:
This code doesnot create empty files as required by the step `Verify Volumes are accessible by creating an empty file on the volume`